lib/easypost/utilities/static_mapper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module EasyPost::InternalUtilities::StaticMapper
+  BY_PREFIX = {
+    'adr' => EasyPost::Models::Address,
```

Comment:
question: Can we import these once and reference them twice instead of needing the entire namespace twice?

Reply:
Do you mean import `require_relative '../models/address'` on the top to save the namespace down below? I don't believe it's possible as Ruby can't resolve it to the correct class.